### ISSUE TITLE
Add accessibility considerations to the HTML "autocomplete" attribute page

### ISSUE THREAD

**Issue by anna-yeddi** [mdn/content#37170]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete

### What information was incorrect, unhelpful, or incomplete?

The accessibility consideration is missing from the page. In some cases, web authors are trying to turn off autocomplete or assign an invalid value to the attribute to disable it, but this is a tool that many disabled users are relying on while filling variety of forms.

### What did you expect to see?

We could mention the importance for `autocomplete` for users with disabilities (both for those who use assistive technology like a screen reader and for those who do not use it). And mention that the lack of autocomplete or `autocomplete=off` and so on is something that automated accessibility tests would reliably catch and flag. This affects the usability of a form specifically but also the accessibility compliance of the website (EN 301 549 as well as EAA and other international, national, state, and local accessibility legislation are usually rely on the WCAG 2.1 and 2.2 which include the Success Criterion 1.3.5 Identify Input Purpose (Level AA)).

### Do you have any supporting links, references, or citations?

https://www.w3.org/WAI/WCAG22/Understanding/identify-input-purpose and its success technique https://www.w3.org/WAI/WCAG22/Techniques/html/H98 include more information that is better phrased than I can write :)

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/html/attributes/autocomplete`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/attributes/autocomplete/index.md
* Last commit: https://github.com/mdn/content/commit/06b418a190b8e4a46682ab706d14984e7db34862
* Document last modified: 2024-09-11T03:36:25.000Z

</details>

**Comment by xopsio** [comment]
I’m planning to add an accessibility considerations section to the autocomplete attribute page, based on WCAG 2.2 SC 1.3.5 and related techniques.
